os/tools/mkromfsimg.sh

```diff
@@ -82,7 +82,6 @@ if [ ! -r $topdir/.config ]; then
     exit 1
 fi
 
-romfsetc=`grep CONFIG_NSH_ROMFSETC= $topdir/.config | cut -d'=' -f2`
```

Comment:
Here are many CONFIG_NSH_XX which are not used in TizenRT. I think we had better cleanup all of content in this file if we don't have any reason to remove this only immediately.